### PR TITLE
✅  TEST: [bulk/main] add tests for file naming + check for middle name

### PR DESCRIPTION
**Pull request by chancesmith**
## Purpose of PR

How will we know if expectations change and we get the right result?

> Legacy code is untested code.

## Changes Made

- test filename functions
- check for middlename

## Screenshots

Screen Shot 2023-01-07 at 7 24 47 AM

### CODE REVIEW

File: csv-bulk-signature/src/main.test.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const contactGenerator: () => Contact = () => ({
+  "Full Name*": "nothing",
+  Credentials: "nothing",
+  "Title*": "nothing",
+  "Office Phone*": "nothing",
+  "Mobile Phone": "nothing",
+  "Calendly Link": "nothing",
+  "Brand*": "ata-capital",
+});
```
Comment (chancesmith) [inline csv-bulk-signature/src/main.test.ts:12]: create contact factory for each test that needs a contact arrangment

File: csv-bulk-signature/src/main.test.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  it("should return file name with a middle name", () => {
+    const contact: Contact = {
+      ...contactGenerator(),
+      "Full Name*": "John Michael Doe",
+    };
+    const result = getFullNameFileName(contact);
+    expect(result.fullNameFileName).toBe("john_michael_doe");
+  });
+});
```
Comment (chancesmith) [inline csv-bulk-signature/src/main.test.ts:33]: test for first+last and first+middle+last

File: csv-bulk-signature/src/main.test.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+  it("should return file name with a middle name", () => {
+    const contact: Contact = {
+      ...contactGenerator(),
+      "Full Name*": "John Michael Doe",
+    };
+    const result = getFileName(contact);
+    expect(result).toBe("jmichaeldoe");
+  });
+});
```
Comment (chancesmith) [inline csv-bulk-signature/src/main.test.ts:54]: test for first+last and first+middle+last

**Review by chancesmith — COMMENTED** [review]
Not many I/O-less items to test, but it's a shoehorn to get tests into the project

File: csv-bulk-signature/src/main.test.ts (hunk 2 above)
Comment (chancesmith) [reply csv-bulk-signature/src/main.test.ts:33]: My pushback here is there isn't a case for middle name yet...but I thought it was worth exposing for how many people have middle names. 😄

**Review by adamnitti — APPROVED** [review]

**Pull request merged**